database/manual-updates/2021-06-16_1-add-personal-relationship-types.md

```diff
@@ -0,0 +1,101 @@
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'parentOfUnbornChild') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'unbornChild');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'siblingOfUnbornChild') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'unbornSibling');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'fosterCarerSupportCarer') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'supportCarerFosterCarer');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'privateFosterCarer') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'privateFosterChild');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'privateFosterChild') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'privateFosterCarer');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'fosterChild') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'fosterCarer');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'supportCarerFosterCarer') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'fosterCarerSupportCarer');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'neighbour') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'neighbour');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'inContactWith') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'inContactWith');
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'acquaintance') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'acquaintance');
+
+-- insert into new personal relationship table
+INSERT INTO dbo.sccv_personal_relationship (fk_person_id, fk_other_person_id, fk_personal_relationship_type_id)
```

Comment:
We have bit of a mismatch between `fk_other_person_id` and the column set in https://hackney.atlassian.net/browse/SCT-409 (`fk_second_person_id`), so just need to tweak one so they match.

Reply:
Ahhh, when we created the manual update for creating the table we changed it to `fk_other_person_id` but forgot to update the Jira ticket, sorry! See https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/295#discussion_r650746315.

Just doubled checked staging to make sure we ran the manual update code and we did, but just noticed a smol thing - the name we gave for the foreign key constraint doesn't match the name of the foreign key i.e. ` "fk_second_person" FOREIGN KEY (fk_other_person_id) REFERENCES dbo.dm_persons(person_id)`.

Will update the manual update so production is consistent.

```
                                         Table "dbo.sccv_personal_relationship"
              Column              |            Type             | Collation | Nullable |             Default
----------------------------------+-----------------------------+-----------+----------+----------------------------------
 id                               | bigint                      |           | not null | generated by default as identity
 fk_person_id                     | bigint                      |           |          |
 fk_other_person_id               | bigint                      |           |          |
 fk_personal_relationship_type_id | bigint                      |           |          |
 sccv_created_at                  | timestamp without time zone |           |          |
 sccv_created_by                  | character varying(300)      |           |          |
 sccv_last_modified_at            | timestamp without time zone |           |          |
 sccv_last_modified_by            | character varying(300)      |           |          |
Indexes:
    "xpk_personal_relationship" PRIMARY KEY, btree (id)
Foreign-key constraints:
    "fk_person" FOREIGN KEY (fk_person_id) REFERENCES dbo.dm_persons(person_id)
    "fk_personal_relationship_type" FOREIGN KEY (fk_personal_relationship_type_id) REFERENCES dbo.sccv_personal_relationship_type(id)
    "fk_second_person" FOREIGN KEY (fk_other_person_id) REFERENCES dbo.dm_persons(person_id)
Referenced by:
    TABLE "dbo.sccv_personal_relationship_detail" CONSTRAINT "fk_personal_relationship" FOREIGN KEY (fk_personal_relationship_id) REFERENCES dbo.sccv_personal_relationship(id) ON DELETE CASCADE
```